app/views/shared/_organization_header.html.erb

```diff
@@ -1,19 +1,19 @@
-<div class="position-relative overflow-hidden mb-5 pb-2 pt-4 bg-light organization border-bottom">
-  <div class="container">
-    <div class="d-flex justify-content-between">
-      <h1 class="display-5 font-weight-normal mb-3">
-        <% if @organization.icon.attached? %>
-          <%= image_tag(@organization.icon, class: 'organization-icon') %>
-        <% end %>
+<% content_for :org_header do %>
+  <div class="position-relative overflow-hidden mb-5 pb-2 pt-4 bg-light organization border-bottom">
+    <div class="container">
+      <div class="d-flex justify-content-between">
+        <h1 class="display-5 font-weight-normal mb-3">
+          <% if @organization.icon.attached? %>
+            <%= image_tag(@organization.icon, class: 'organization-icon') %>
+          <% end %>
 
-        <%= link_to_unless controller_name == 'organizations', @organization.name, @organization, class: 'text-body' %>
-      </h1>
+          <%= link_to_unless controller_name == 'organizations', @organization.name, @organization, class: 'text-body' %>
+        </h1>
 
-      <% if controller_name == 'organizations' %>
-        <%= link_to 'Edit', edit_organization_path(@organization), class: 'align-self-center btn btn-primary' if can? :edit, @organization %>
-      <% end %>
+        <% if controller_name == 'organizations' %>
+          <%= link_to 'Edit', edit_organization_path(@organization), class: 'align-self-center btn btn-primary' if can? :edit, @organization %>
+        <% end %>
+      </div>
     </div>
-
-    <%= yield %>
```

Comment:
I think you still need `yield` here so that content passed to the `render 'shared/organization_header'` blocks will appear, such as here:

https://github.com/pod4lib/aggregator/blob/8c1878467bcc5112b51ca05a4655930fd6635f70/app/views/organizations/show.html.erb#L1

Reply:
@corylown Tested it out and you're right, the yield has been added back in. I'm not sure if this is standard for you, but this may be a good PR for a squash-and-merge so as not to clutter up the project history with a bunch of commits that were ultimately superfluous.